### PR TITLE
Heatmap

### PR DESCRIPTION
**Pull request by stuartskelton**
Ok I have my heat map module,

if you accept the #20 PR, I will update the heatmap module to reflect this.

### CODE REVIEW

File: lib/SVG/TT/Graph/HeatMap.pm [hunk 1]
```diff
@@ -0,0 +1,929 @@
...
+    'title' => 'Disk',
+  });
+
+  print "Content-type: image/svg+xml\n\n";
+  print $graph->burn();
+
+=head1 DESCRIPTION
+
+This object aims to allow you to easily create high quality
+SVG line graphs of XY data. You can either use the default style sheet
```
Comment (ranguard) [inline lib/SVG/TT/Graph/HeatMap.pm]: not a line graph

**Comment by ranguard** [comment]
Minor comment, haven't run the code yet but if you make the changes as per #20 then I'll merge and release :) - Thanks so much

File: lib/SVG/TT/Graph/HeatMap.pm (hunk 1 above)
Comment (stuartskelton) [inline lib/SVG/TT/Graph/HeatMap.pm]: and it is neither a XY plot, I will have to work on the documentation before I would even feel good about letting this go up to CPAN. (sorry)

**Comment by stuartskelton** [comment]
I think this is all good now.  I have another version of the heat maps down the line but I would rather this version get included into a release.

**Comment by ranguard** [comment]
hi @stuartskelton could you have a look at the bubble script please:
```
perl -I lib script/synopsis_bubble.pl
Can't use string ("10") as a HASH ref while "strict refs" in use at lib/SVG/TT/Graph.pm line 508.
```

I'll release a new version once that's sorted.

Thanks

Leo

**Comment by stuartskelton** [comment]
OK. I'll look at it soon.

**Comment by ranguard** [comment]
Thanks for all your work on this... it's much appreciated

**Comment by stuartskelton** [comment]
I have tracked this down, Ceil is used in Graph.pm and its removal because of #20 causes this to fail. 

I have created a PR #22 to fix this.

**Pull request merged**